DRA: core update for 1.34 (#51483)
kubernetes/website: content/en/docs/tasks/configure-pod-container/assign-resources/set-up-dra-cluster.md

<!-- steps -->

> ## Optional: enable additional DRA API groups {#enable-dra}

Reviewer: ```suggestion
## Optional: enable legacy DRA API groups {#enable-dra}
```
?

> DRA reached GA in Kubernetes 1.34 and is enabled by default.

Reviewer: ```suggestion
DRA graduated to stable in Kubernetes 1.34 and is enabled by default.
```

```
error: the server doesn't have a resource type "deviceclasses"
```

Try the following troubleshooting steps:

1. Reconfigure and restart the `kube-apiserver` component.

> 1. If the complete `pod.spec.resourceClaims` field gets removed from Pods or

Reviewer: ```suggestion
1. If the complete `.spec.resourceClaims` field gets removed from Pods, or if
```

   Pods get scheduled without considering the ResourceClaims, then verify

>    that the `DynamicResourceAllocation` feature gate is not turned off

Reviewer: ```suggestion
   that the `DynamicResourceAllocation` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) is not turned off
```
In GA quality docs, we **always** put in this hyperlink.
